Jenkinsfile

```diff
@@ -1,117 +1,12 @@
-@Library('socrata-pipeline-library')
+@Library('socrata-pipeline-library@v0.0.1')
```

Comment:
Missing an underscore?

```
/* Using a version specifier, such as branch, tag, etc */
@Library('my-shared-library@1.0') _
```

Reply:
The underscore wasn't the problem.  It turns out it was `jgit`, which is a redundant and unnecessary version of git supplied by Jenkins.  It is working now.